src/avx2/stage1.rs

```diff
@@ -345,9 +372,18 @@ fn finalize_structurals(
     structurals
 }
 
-//WARN_UNUSED
-/*never_inline*/
-//#[inline(never)]
+pub fn find_bs_bits_and_quote_bits(v: __m256i) -> ParseStringHelper {
```

Comment:
we should make sure this is inlined when called and perhaps investigate if the creation of a struct comes with a performance penalty.